aws-cfn/cross-service/resilient-workflow/README.md

```diff
@@ -0,0 +1,240 @@
+
+To access the load balancer endpoint, you must allow inbound traffic
+on port 80 from your computer's IP address to your VPC. If this rule doesn't exist, the
+example tries to add it. Alternately, you can
+[add a rule to the default security group for your VPC](https://docs.aws.amazon.com/vpc/latest/userguide/security-group-rules.html)
+and specify your computer's IP address
+as a source.
+
+This examples is deployed as an AWS CloudFormation stack. This can be achieved
+using the [AWS CloudFormation Console](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-console.html),
+or using the [AWS Command Line Interface](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-cli.html) (AWS CLI).
+This guide presents step-by-step instructions using the AWS CLI.
+
+### Instructions
+
+#### Build and manage a resilient service
+
+##### Deploy resources
+
+Use AWS CloudFormation to create the following AWS resources:
+
+1. A DynamoDB table that acts as a service that recommends books, movies, and songs.
+2. An instance profile and an associated role and policy that grants permission to
+   instances to access DynamoDB and Systems Manager.
+3. A launch template that specifies the instance profile and a startup script
+   that starts a Python web server on each instance.
+4. An Auto Scaling group that starts EC2 instances, one in each of three
+   Availability Zones.
+5. An Application Load Balancer that handles HTTP requests to a single endpoint.
+6. A target group that connects the load balancer to instances in the Auto Scaling group.
+7. A listener that is added to the load balancer and forwards requests to the target group.
+
+To deploy these resources, you will need to provide several pieces of information about your AWS Account.
+
+| Parameter | AWS CLI                                                                                                      |
+| --------- | ------------------------------------------------------------------------------------------------------------ |
+| `VPCId`   | `aws ec2 describe-vpcs --filters Name=is-default,Values=true --query "Vpcs[0].VpcId"`                        |
+| `SGId`    | `aws ec2 describe-security-groups --group-names default --query "SecurityGroups[0].GroupId"`                 |
+| `Subnets` | `aws ec2 describe-subnets --filters Name=vpc-id,Values="vpc-0abcdef1234567890" --query "Subnets[].SubnetId"` |
+
+After running these commands, replace the placeholders in `params.json` with the returned values.
+
+```
+aws cloudformation create-stack \
+  --stack-name resilient-service \
+  --template-body file://./resilient-service.yaml \
+  --parameters: file://./params.json \
+  --capabilities CAPABILITY_NAMED_IAM
+```
+
+After creating the stack, you can find the URL for the Load Balancer in the stack's parameters.
+
+```
+aws cloudformation describe-stacks --stack-name resilience-demo --query 'Stacks[0].Outputs[?OutputKey==`LB`].OutputValue | [0]'
```

Comment:
Need double quotes on Windows.

Reply:
Done

Reply:
Undone and added a note (because "`LB`" tries to execute LB on bash... you can never win...)